pmd-java/src/test/java/net/sourceforge/pmd/lang/java/oom/MetricsForceHook.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.oom;
+
+/**
+ * Provides a hook into package-private methods of {@code oom}.
+ *
+ * @author Clément Fournier
+ */
+public class MetricsForceHook {
```

Comment:
Just FYI - maybe this could also be solved via a custom [JUnit Rule](https://github.com/junit-team/junit4/wiki/rules). It would set the force flag before the test and reset it after again....